packages/google_sign_in/google_sign_in_web/lib/google_sign_in_web.dart

```diff
@@ -28,14 +38,52 @@ class GoogleSignInPlugin {
 
   Future<dynamic> handleMethodCall(MethodCall call) async {
     // Await for initialization promises to complete, then do your thing...
-    await isJsLoaded;
+    await _isGapiInitialized;
 
-    html.window.console.log('Doing things! $call');
+    html.window.console.debug(call);
+
+    final GoogleAuth authInstance = gapi.auth2.getAuthInstance();
+    final GoogleUser currentUser = authInstance?.currentUser?.get();
 
     switch (call.method) {
-      case 'init':
-        // Initialize the gapi
-        return _init(call.arguments);
+      case 'init': // void
+        return _init(call.arguments)
```

Comment:
you should be able to just return `null`

Reply:
I think returning null breaks [this logic](https://github.com/flutter/plugins/blob/4eeb744bfce3cbacd64dcce32b52295c3299fe89/packages/google_sign_in/lib/google_sign_in.dart#L233-L238), but that's what the java implementation is doing, ssssoooo null it is!

Reply:
Hmm I think you're right about breaking that null-check logic. Maybe return `true` or something? Just wary of the `toString()` call just to return a signal that the initialization was successful

Reply:
I like return true, I'll get that done. `.toString` rubs me the wrong way too.

However, the [Java implementation](https://github.com/flutter/plugins/blob/master/packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java#L252-L254) returns a success-with-null; I guess null is being handled differently, OR the plugin itself makes sure not to call init multiple times.